tezos/encoding-derive/src/enc.rs

```diff
@@ -0,0 +1,137 @@
+use crate::encoding::*;
```

Comment:
@akoptelov 
please, make sure, that all new files contains:
```
// Copyright (c) SimpleStaking and Tezedge Contributors
// SPDX-License-Identifier: MIT
```

Reply:
Done.